loadtest/store/memstore/store.go

```diff
@@ -396,7 +475,15 @@ func (s *MemStore) SetChannelMembers(channelMembers *model.ChannelMembers) error
 		if s.channelMembers[cm.ChannelId] == nil {
 			s.channelMembers[cm.ChannelId] = make(map[string]*model.ChannelMember)
 		}
-		s.channelMembers[cm.ChannelId][cm.UserId] = cm
+
+		c := s.channelMembersQueue.Get().(*model.ChannelMember)
```

Comment:
A quick one-liner before these sections would be helpful for future readers to clarify what this is doing. Something like 
`// We check if the tail of the queue is already full. And if so, we delete the item from the map. This is done to keep the number of items in the map equal to the queue size and prevent unbounded memory growth.`